Cargo.toml

```diff
@@ -24,8 +24,14 @@ core-foundation-sys = "0.8.3"
 winapi = { version = "0.3.9", features = ["activation", "combaseapi", "objbase", "roapi", "winerror", "winstring"] }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-js-sys = "0.3.46"
+js-sys = "0.3.50"
```

Comment:
if I had to take a guess at why this is required, `cxx` probably has a different minimum version of `syn` or some other proc macro crate which doesn't play nice with earlier versions of `js-sys`.

Reply:
Thanks. Yes, something like that seems likely.